handlers/main.yml

```diff
@@ -1,5 +1,6 @@
 - name: restart docker
-  become: true
+  become: "{{ docker_become }}"
+  become_user: "{{ docker_become_user }}"
   service:
     name: docker
     service: restarted
```

Comment:
Most, if not all, the oasis roles usually put the `become` and `become_user` options after the call to the module. Do we want to maintain that sort of format or is it not a topic of concern?

Reply:
There is no such specification in the OASIS standards. Most of the roles I authored do not (I prefer the order to be name, become, become_user, <module>, args, then follow up with when/register/loop/until after that), many of the roles authored by others do.

Ultimately, I don't think it matters too much, although it's nice to at least keep it the same inside of each tasks file.